### PR TITLE
SFT-133 Elysia page

### PR DESCRIPTION
**Pull request by JensVarughese**
The Elysia Page with a basic implementation of a gallery. The learn more button on the cars page is also linked to the Elysia page.

### CODE REVIEW

File: src/app/elysia/elysia.component.css
Comment (liconway) [inline src/app/elysia/elysia.component.css:69]: If you can put a new line after here, just to appease the Git gods

**Review by liconway — APPROVED** [review]
Lgtm :+1: We'll just need to make sure the Elysia gallery images are higher quality for release.

I'd wait for @JaydenBischoff's approval before merging though

**Review by JaydenBischoff — CHANGES_REQUESTED** [review]
Looks really good but I noticed that the pictures in the gallery are really blurry. Were the pictures really small on the original website or could you try resaving and adding the photos to the assets folder.

**Comment by JensVarughese** [comment]
> Looks really good but I noticed that the pictures in the gallery are really blurry. Were the pictures really small on the original website or could you try resaving and adding the photos to the assets folder.

The pictures on the website were only 100x100 pixels. We could change them when we officially deploy the website.

**Review by JaydenBischoff — APPROVED** [review]
Okay sure we can change them later. Everything else looks good

**Pull request merged**
